GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -817,13 +817,13 @@ public void ReloadTranslation()
             Translator.Translate(this, AppSettings.CurrentTranslation);
         }
 
-        private void ShowLoading(bool sync = true)
+        private void ShowLoading(bool showSpinner = true)
         {
-            _loadingControlSync.Visible = sync;
+            _loadingControlSync.Visible = showSpinner;
             _loadingControlSync.Left = (ClientSize.Width - _loadingControlSync.Width) / 2;
             _loadingControlSync.Top = (ClientSize.Height - _loadingControlSync.Height) / 2;
 
-            _loadingControlAsync.Visible = !sync;
+            _loadingControlAsync.Visible = !showSpinner;
```

Comment:
_loadingControlSync is the spinner shown at creation and when refreshing until first revision
_loadingControlAsync is the "Loading" text while still revisions are read

So showSpinner makes sense, renaming the controls

Reply:
Added documentation

The Async/Text "Loading..." was not shown since some time ago

Some other refresh, obviously waiting some 20h more before squashing for proper review